ci/build_python.sh

```diff
@@ -20,8 +20,29 @@ CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
 
 sccache --zero-stats
 
-# This calls mambabuild when boa is installed (as is the case in the CI images)
-RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION) rapids-conda-retry mambabuild -c "${CPP_CHANNEL}" conda/recipes/rmm
+RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION)
+export RAPIDS_PACKAGE_VERSION
+
+# rattler-build doesn't have built-in support for GIT_DESCRIBE_NUMBER and
+# GIT_DESCRIBE_HASH so we set them in the environment first
+# shellcheck disable=SC2034
+IFS=- read -r TAG_VERSION GIT_DESCRIBE_NUMBER GIT_DESCRIBE_HASH <<< "$(git describe --tags)"
+unset TAG_VERSION
+export GIT_DESCRIBE_NUMBER
+export GIT_DESCRIBE_HASH
+
+mamba install rattler-build -c conda-forge -y
+
+rattler-build build --recipe conda/recipes/rmm \
+                    --experimental \
+                    --no-build-id \
+                    --channel-priority disabled \
+                    --output-dir "$RAPIDS_CONDA_BLD_OUTPUT_DIR" \
+                    -c "${CPP_CHANNEL}" \
+                    -c rapidsai \
+                    -c rapidsai-nightly \
+                    -c conda-forge \
+                    -c nvidia
```

Comment:
As mentioned above, adding `${CPP_CHANNEL}` adds an implicit `--override-channels` so we need to list them inline

Reply:
I'll report it upstream -- it feels like a bug, but it might be a conscious break from some implicit `conda` behavior, but in that case, documentation is certainly merited.

Reply:
So this turns out to be a misinterpretation by me -- the `librmm` build works without channel specifications because all of the dependencies are available on `conda-forge` (i guess?).  `rattler-build` doesn't support reading channels from `condarc` files at all and just defaults to `conda-forge` if nothing is specified.

There's an open PR to add channels to the variant configuration file, but currently the only supported method of channel specification is via CLI.

https://github.com/prefix-dev/rattler-build/pull/1416